db/people.yml

```diff
@@ -464,22 +464,44 @@ cbullard:
   active: true
   aka: []
   avatar: https://github.com/chloeann95.png
-  bio: bio
-  education: []
-  email: cab2344@columbia.edu
+  bio: Chloë grew up in Michigan and attended Michigan State University for her undergraduate
+    degrees in Physics and Mechanical Engineering, where she worked on computational plasma
+    physics research. Afterward she attended the University of Toledo completing a Masters
+    in Chemical Engineering while working on modeling CFD simulations for biomass with NREL.
+    When not working you can find her looking for a new game to play or a new song to learn 
+    on the piano.
+  education:
+    - begin_year: 2013
+      begin_month: 9
+      degree: B.S. in Physics
+      degree: B.S. in Mechanical Engineering
```

Comment:
Will do, do you want me to go back and reply to previous comments or leave them be and just address future comments?

Reply:
all.   The general goal is to reduce entropy and get things to finished.....it means removing uncertainty.  Generally a bit of extra typing can do this very well.  For example, now we are having a conversation that is taking more time than the original task....  Its ok, because it will pay off big dividends in the future, a learning moment, but the general guidance is, before sending something back, spend some time thinking about what could you do that would remove the most entropy and uncertaintly....